### PR TITLE
Add clippy check operation of failpoint test

### PR DESCRIPTION
**Pull request by yuqi1129**
Signed-off-by: yuqi1129 <yuqi4733@gmail.com>

### CODE REVIEW

**Comment by yuqi1129** [comment]
@tabokie Can you take time to review if this PR is necessary?

**Comment by tabokie** [comment]
You can add a simple Makefile, the script doesn't look very professional.

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Contributions are always welcome! Here are a few tips for making a PR:
 ```
 cargo fmt --all -- --check
 cargo clippy --all --all-targets -- -D clippy::all
+cargo clippy --features failpoints --all --all-targets -- -D clippy::all
```
Comment (tabokie) [inline README.md:70]: merge them into `--all-features`

Comment (yuqi1129) [reply README.md:70]: 👌

**Comment by yuqi1129** [comment]
> You can add a simple Makefile, the script doesn't look very professional.

Got it, To add a make file just for code format and checks seems to be unnecessary, I will delete the shell script later, this issue only update the readme doc

**Review by tabokie — APPROVED** [review]

**Pull request merged**